### DISCUSSION TITLE
Is it going to support selecting qualities for mp4?

### DISCUSSION THREAD

**Discussion by bgr11n** [vidstack/player#1089] [Q&A] [answered]
Is it going to support selecting qualities for sources other than hls, for example multiple mp4 files, etc?

Thank you!

**Comment by mihar-22** [comment, marked as answer]
No plans to support it just yet and we strongly recommend HLS for many reasons around user experience. We have a free tool we're going to release soon that will help with taking your MP4 and creating a HLS stream. Most don't realize you can static host HLS files just like MP4 files!

**Reply by bgr11n** [reply]
@mihar-22 Got it, thank you!
